tests/wallet/nft_wallet/test_nft_wallet.py

```diff
@@ -212,7 +212,7 @@ async def test_nft_wallet_rpc_creation_and_list(two_wallet_nodes: Any, trusted:
         {
             "wallet_id": nft_wallet_0_id,
             "artist_address": ph,
-            "hash": 0xD4584AD463139FA8C0D9F68F4B59F184,
+            "hash": "0xD4584AD463139FA8C0D9F68F4B59F184",
```

Comment:
I don't understand your rationale for changing this from a (compact and efficient) binary representation to a (slow and inflated) ascii string representation.

Even if there's a really good reason to represent this hash in ascii-form, surely you wouldn't need the `0x` prefix.

Reply:
I may be misunderstanding something actually, I was thinking that this was part of the smart contract, but this is the API to our RPC client, right? So this isn't really in a place where we have the ambition to be efficient, since we already use JSON at this level. And presumably the client class will turn this hash into a string when encoding it as JSON anyway.

However, I think there's a consistency argument to be made. The "artist_address" is a puzzle hash, fundamentally the same type as this "hash". They should probably use the same representation

Reply:
That's correct, the tests are bypassing the normal JSON serialization/deserialization that would be applied in `RpcServer`, so the onus is on the test writer to ensure that the requests/responses are JSON compatible. In this case, the hash(es) being supplied as raw bytes is incorrect, as that isn't directly possible with JSON. Ideally we should come up with a test fixture (or something) that can wrap an RPC API object instance and enforce JSON compatibility.

I agree that we should be consistent, using strings to pass around hashes (puzzle hashes, data hashes, etc.)